Dockerfile.py2

```diff
@@ -2,6 +2,7 @@ FROM python:2.7.15-slim-stretch
 
 RUN pip install virtualenv
 RUN mkdir /venv && virtualenv --python=python2 /venv
-COPY py2-requirements.txt /
-RUN /venv/bin/pip install -r /py2-requirements.txt
+COPY py2-requirements.txt /requirements.txt
+COPY settings.yml /settings.yml
```

Comment:
the `settings.yml` file should be visible at runtime, as the whole folder is mounted as a volume. I wouldn't bake it into the image unless it was influencing the result e.g. what is in the virtualenv?

Reply:
settings.yml is a generated file, it's absence is breaking your build.

Reply:
:+1: for providing it, but as a mounted volume like the rest of the files not related to the venv